internal/server/server.go

```diff
@@ -604,32 +638,26 @@ func (s *Service) registerDetectorComponent() {
 
 // registerHelmComponent registers helm tools if enabled
 func (s *Service) registerHelmComponent() {
-	if s.cfg.AdditionalTools["helm"] {
-		logger.Debugf("Registering Kubernetes tool: helm")
-		helmTool := helm.RegisterHelm()
-		helmExecutor := k8s.WrapK8sExecutor(helm.NewExecutor())
-		s.mcpServer.AddTool(helmTool, tools.CreateToolHandler(helmExecutor, s.cfg))
-	}
+	logger.Debugf("Registering Kubernetes tool: helm")
```

Comment:
More like a debug log, or we have somewhere else tells the enabled tool list?

Reply:
yes, it is debug log only for user to read. AI need to call grpc "tools/list" to need the tool list